File: 01FlutterApps/middle_layer_springboot/knpiano_java/src/main/java/com/liu/springboot04web/bean/Kn02f005FeeMonthlyReportBean.java
package com.liu.springboot04web.bean;

public class Kn02f005FeeMonthlyReportBean implements KnPianoBean {

    protected float     shouldPayLsnFee;
    protected float     hasPaidLsnFee;
    protected float     unpaidLsnFee;
    protected String    lsnMonth;
    protected String    stuName;
    protected String    stuId;

    public float getShouldPayLsnFee() {
        return shouldPayLsnFee;
    }
    public void setShouldPayLsnFee(float shouldPayLsnFee) {
        this.shouldPayLsnFee = shouldPayLsnFee;
    }
    public float getHasPaidLsnFee() {
        return hasPaidLsnFee;
    }
    public void setHasPaidLsnFee(float hasPaidLsnFee) {
        this.hasPaidLsnFee = hasPaidLsnFee;
    }
    public float getUnpaidLsnFee() {
        return unpaidLsnFee;
    }
    public void setUnpaidLsnFee(float unpaidLsnFee) {
        this.unpaidLsnFee = unpaidLsnFee;
    }
    public String getLsnMonth() {
        return lsnMonth;
    }
    public void setLsnMonth(String lsnMonth) {
        this.lsnMonth = lsnMonth;
    }
    public String getStuName() {
        return stuName;
    }
    public void setStuName(String stuName) {
        this.stuName = stuName;
    }
    public String getStuId() {
        return stuId;
    }
    public void setStuId(String stuId) {
        this.stuId = stuId;
    }

}

File: 01FlutterApps/front_dart_flutter/lib/02LsnFeeMngmnt/Kn02f005FeeMonthlyReportBean.dart
class Kn02f005FeeMonthlyReportBean {
  final double shouldPayLsnFee;
  final double hasPaidLsnFee;
  final double unpaidLsnFee;
  final String lsnMonth;
  final String stuName;
  final String stuId;

  Kn02f005FeeMonthlyReportBean({
    required this.shouldPayLsnFee,
    required this.hasPaidLsnFee,
    required this.unpaidLsnFee,
    required this.lsnMonth,
    required this.stuName,
    required this.stuId,
  });

  factory Kn02f005FeeMonthlyReportBean.fromJson(Map<String, dynamic> json) {
    return Kn02f005FeeMonthlyReportBean(
      shouldPayLsnFee : json['shouldPayLsnFee']?.toDouble() ?? 0.0,
      hasPaidLsnFee   : json['hasPaidLsnFee']?.toDouble() ?? 0.0,
      unpaidLsnFee    : json['unpaidLsnFee']?.toDouble() ?? 0.0,
      lsnMonth        : json['lsnMonth'] as String ? ?? '',
      stuName         : json['stuName'] as String ? ?? '',
      stuId           : json['stuId'] as String ? ?? '',
    );
  }
}